api/src/main/java/org/apache/iceberg/RowDelta.java

```diff
@@ -47,12 +47,12 @@ public interface RowDelta extends SnapshotUpdate<RowDelta> {
   RowDelta addDeletes(DeleteFile deletes);
 
   /**
-   * Delete a {@link DataFile} from the table.
+   * Remove a {@link DataFile} from the table.
    *
    * @param file a data file
    * @return this for method chaining
    */
-  default RowDelta deleteFile(DataFile file) {
+  default RowDelta removeRows(DataFile file) {
```

Comment:
@talatuyarer I think something is wrong with this PR.  Did you get other commits in the same PR?  There are a number of what look like unrelated changes.